test/jdk/java/awt/print/PrinterJob/ImagePrinting/AlphaPrintingOffsets.java

```diff
@@ -41,44 +38,58 @@
 import javax.print.attribute.standard.Sides;
 import javax.swing.JFrame;
 
+import static java.awt.print.PageFormat.LANDSCAPE;
+import static java.awt.print.PageFormat.PORTRAIT;
+import static java.awt.print.PageFormat.REVERSE_LANDSCAPE;
+
 /*
  * @test
  * @bug 8307246
  * @key printer
  * @library ../../../regtesthelpers
  * @build PassFailJFrame
- * @summary Test for comparing offsets of images drawn with opaque and translucent colors printed in all orientations
+ * @summary Test for comparing offsets of images drawn with opaque
+ *          and translucent colors printed in all orientations
  * @run main/manual AlphaPrintingOffsets
  */
 
 public class AlphaPrintingOffsets {
     private static final String INSTRUCTIONS =
-                    "Tested bug occurs only on-paper printing so you mustn't use PDF printer\n\n" +
-                    "1.Java print dialog should appear.\n" +
+            "Tested bug occurs only on-paper printing " +
+                    "so you mustn't use PDF printer\n\n" +
+                    "1. Java print dialog should appear.\n" +
```

Comment:
```suggestion
                    "Tested bug occurs only on-paper printing " +
                    "so you mustn't use PDF printer\n\n" +
                    "1. Java print dialog should appear.\n" +
```
Alternatively, reduce the indentation of *all* the lines with the instructions.